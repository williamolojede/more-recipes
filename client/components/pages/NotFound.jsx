import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
  <div className="page page__not-found">
    <main>
      <h1>404</h1>
      <p>Sorry, the web page your are looking for is missing
        <span role="img" aria-label="cry">ðŸ˜­</span>
      </p>
      <Link to="/">Go back to the home page</Link>
    </main>
  </div>
);


export default NotFound;
