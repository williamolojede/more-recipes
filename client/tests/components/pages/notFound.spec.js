import React from 'react';
import NotFound from '../../../components/pages/NotFound.jsx';

describe('Manage Recipes', () => {
  it('should render with right amount of elements', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
