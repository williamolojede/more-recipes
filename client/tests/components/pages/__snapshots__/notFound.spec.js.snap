// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Manage Recipes should render with right amount of elements 1`] = `
<div
  className="page page__not-found"
>
  <main>
    <h1>
      404
    </h1>
    <p>
      Sorry, the web page your are looking for is missing
      <span
        aria-label="cry"
        role="img"
      >
        ðŸ˜­
      </span>
    </p>
    <Link
      replace={false}
      to="/"
    >
      Go back to the home page
    </Link>
  </main>
</div>
`;
